types/steam-tradeoffer-manager/index.d.ts

```diff
@@ -10,8 +10,8 @@ import SteamUser = require('steam-user');
 import SteamCommunity = require('steamcommunity');
 import CEconItem = require('steamcommunity/classes/CEconItem');
 import FileManager = require('file-manager');
-
 import TradeOffer = require('./lib/classes/TradeOffer');
+import type { InventoryCallback, EResultError } from './common';
```

Comment:
Instead of mixing the import styles I'd really prefer having these types live in the namespace.

Reply:
Ok